.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     # testing of PR format
     - env:
        - DESC="test Issue ref in PR description"
-       - CMD="export PULL_REQUEST=$TRAVIS_PULL_REQUEST && ./.ci/travis/travis.sh pr-description"
+       - CMD="export PULL_REQUEST=$TRAVIS_PULL_REQUEST && ./.ci/travis/xtr_pr-description.sh"
```

Comment:
this job is moved to github actions, shouldnt it be removed from travis config?

Reply:
fixed by rebase.